### PR TITLE
New testnet owner

### PR DESCRIPTION
**Pull request by justinjmoses**
Migrating to new testnet owner

## Migration
- [x] Kovan
- [x] Kovan OVM

### CODE REVIEW

File: contracts/test-helpers/TestableMinimalProxyFactory.sol [hunk 1]
```diff
@@ -2,7 +2,6 @@ pragma solidity ^0.5.16;
 
 import "../MinimalProxyFactory.sol";
 
-
```
Comment (justinjmoses) [inline contracts/test-helpers/TestableMinimalProxyFactory.sol:5]: formatting fix

File: publish/src/commands/owner.js [hunk 2]
```diff
@@ -218,7 +220,7 @@ const owner = async ({
 				const tx = await web3.eth.sendTransaction({
 					from: account,
 					to: target,
-					gasPrice,
+					gasPrice: w3utils.toWei(gasPrice, 'gwei'),
```
Comment (justinjmoses) [inline publish/src/commands/owner.js:223]: driveby fix

File: publish/src/commands/owner.js [hunk 3]
```diff
@@ -307,7 +309,7 @@ const owner = async ({
 					const tx = await web3.eth.sendTransaction({
 						from: account,
 						to: deployedContract.options.address,
-						gasPrice,
+						gasPrice: w3utils.toWei(gasPrice, 'gwei'),
```
Comment (justinjmoses) [inline publish/src/commands/owner.js:312]: driveby fix

**Review by ajsantander — APPROVED** [review]

**Review by i-stam — APPROVED** [review]

**Review by jacko125 — APPROVED** [review]
updated testnet owner keys on my side
